### DISCUSSION TITLE
Projektdetails Berechtigung 1.15.3 -> 1.30.0 -> 2.31.

### DISCUSSION THREAD

**Discussion by jens-2305** [kimai/kimai#3700] [Questions & Answers] [answered]
Hallo,

ich habe folgendes Problem wenn ich von Version 1.15.3 auf 1.30.0 Update.

Unter Version 1.15.3 konnte ein "Teamleiter" die Projektdetails seiner Projekte über den "Details" (Graphen Icon) Button
bei seinen Projekten sehen. Nach dem Update auf Version 1.30.0 geht dies nur wenn ich ihm das 
Recht [Project (Admin)] details_project gebe. Dann bekommt er es allerdings bei allen Projekten angezeigt.
Unter Version 1.15.3 war das Recht [Project (Teamlead)] details_teamlead_project aureichend.

Mache ich was falsch, habe ich was übersehen ?

Danke Grüße Jens

**Comment by kevinpapst** [comment]
> Mache ich was falsch

Likely

Not saying, that nothing has changed, but if I would get one cent for every "permission does not work" post, I could buy a huge iceream...

> ein Teamleiter

Way to less information... that does not say anything.

Please create a setup at the demo site and post your findings here, so I can have a look as well.

**Comment by jens-2305** [comment]
I build the example on 

https://demo-stable.kimai.org/


the created the Project "gbs Project 1" user "gbs.team" didn't see the Projectdetails button on the Projectpage and also he can not see the Projectdetails on the Report page. He see's the Projectdetails only when I give the Teamlead Role the right [Project (Admin)] details_project.

**Reply by kevinpapst** [reply]
password?

**Reply by jens-2305** [reply]
gbsstart01

**Comment by kevinpapst** [comment]
Okay, I understand the problem.
There was some back and forth with permissions for that screen. 
The real problem ist that visibility of that report is bound to `details_project`. 
Which then falsly allows to see details of non-allowed projects(e.g. the user can see them to use for time-tracking).

I think the only solution is to bind the visibility of that screen to an entirely new permission `report_project_details`.

**Reply by jens-2305** [reply]
Thanks for your response. Is it possible to fix/implement in a near release ?

**Comment by jens-2305** [comment]
Hello, we now wanted to switch to version 2.31, the behavior of the permission “details_teamlead_project” does not seem to have changed, or rather I am not able to call up the project details when I set this permission for the teamlead. This only works if the right “details_project” (Project Admin) is assigned to the teamlead role. However, every teamlead can then see the details of every project even if they are not assigned to this project.

**Reply by kevinpapst** [reply, marked as answer]
Please create a bug
